tests/units/springer/test_parser.py

```diff
@@ -0,0 +1,346 @@
+                    {
+                        "organization": "Universit\u00e9 Paris-Saclay",
+                        "value": "Centre de Physique Th\u00e9orique, \u00c9cole polytechnique, CNRS, "
+                        "Universit\u00e9 Paris-Saclay, Palaiseau, 91128, France",
+                        "country": "France",
+                    },
+                    {
+                        "organization": "Universit\u00e9 Paris-Saclay",
+                        "value": "IRFU, CEA, Universit\u00e9 Paris-Saclay, Gif-sur-Yvette, 91191, France",
+                        "country": "France",
+                    },
+                ],
+                "surname": "Trawi\u0144ski",
+                "given_names": "Arkadiusz",
+                "email": "Arkadiusz.Trawinski@cea.fr",
+            },
+        ],
+    )
+    for authors, parsed_article in zip(expected_results, parsed_articles):
+        assert authors == parsed_article["authors"]
+
+
+def test_title(parsed_articles):
+    """Test extracting title."""
+    titles = (
+        "Symmetry breaking in quantum curves and super Chern-Simons matrix models",
+        "Revisiting the mechanical properties of the nucleon",
+        "A strategy for a general search for new phenomena using data-derived signal regions and its "
+        "application within the ATLAS experiment",
+    )
+    for title, article in zip(titles, parsed_articles):
+        assert "title" in article
+        assert article["title"] == title
+
+
+def test_date_published(parsed_articles):
+    """Test extracting date_published."""
+    dates_published = ("2019-01-28", "2019-01-29", "2019-02-06")
+    for date_published, article in zip(dates_published, parsed_articles):
+        assert "date_published" in article
+        assert article["date_published"] == date_published
+
+
+def test_license(parsed_articles):
+    """Test extracting license information."""
+    expected_licenses = (
+        [
+            {
+                "license": "CC-BY-3.0",
+                "url": "https://creativecommons.org/licenses/by/3.0",
+            }
+        ],
+        [
+            {
+                "license": "CC-BY-4.0",
+                "url": "https://creativecommons.org/licenses//by/4.0",
+            }
+        ],
+        [
+            {
+                "license": "CC-BY-4.0",
+                "url": "https://creativecommons.org/licenses//by/4.0",
+            }
+        ],
+    )
+    for expected_license, article in zip(expected_licenses, parsed_articles):
+        assert "license" in article
+        assert article["license"] == expected_license
+
+
+def test_dois(parsed_articles):
+    """Test extracting dois."""
+    dois = (
+        "10.1007/JHEP01(2019)210",
+        "10.1140/epjc/s10052-019-6572-3",
+        "10.1140/epjc/s10052-019-6540-y",
+    )
+    for doi, article in zip(dois, parsed_articles):
+        assert "dois" in article
+        assert article["dois"] == doi
+        break
```

Comment:
why do you need break?

Reply:
Good question, I copied pasted without thinking much apparently 🙃 
Gonna remove this

Reply:
Removed